<launch>
  <arg name="robot_name" default="$(optenv ROBOT_NAME recbot0)" />
  <arg name="atak_polygon_topic"  default="zone_from_atak" />
  <arg name="search_polygon_topic" default="/search_polygon/specification" />
  <arg name="zone_id"  default="0" />  

  <group ns="$(arg robot_name)">

    <node name="convert_to_search_polygon" pkg="waypoint_planner" type="convert_to_search_polygon.py" output="screen">
      <param name="robot_name" type="string" value="$(arg robot_name)" />
      <param name="atak_polygon_topic" value="$(arg atak_polygon_topic)" />
      <param name="search_polygon_topic" value="$(arg search_polygon_topic)" />
      <param name="zone_id" value="$(arg zone_id)" />

      <!-- Load common configuration parameters -->
      <rosparam command="load" file="$(find waypoint_planner)/config/default_parameters.yaml" subst_value="true" />

    </node>
  </group>

</launch>
